### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by imon-H** [bromite/bromite#1619] [Ideas]
Is there any possibility to add translation feature like chrome in future build?

**Comment by uazo** [comment]
I speak on my behalf, apart from the technical aspects (which I still don't know, but maybe I'll go and see), I would like to understand the use cases, since this request has appeared several times.
why do you (and others) feel the need for this feature?
let me explain better, there is already a way to translate a site through external services (such as the google translator, but also others), why should it be integrated into the browser?

moreover, the request for integrated translation would mean granting one's own navigation, which has a strong impact on privacy. why do you think a project like bromite should develop this? with such a choice the user would feel authorized to use it.

thanks if you want to answer me.

**Reply by wchen342** [reply]
There is https://browser.mt/ which is client side from the documentations but you will need to audit the code. Plus, embedding such a system is non trivial so I think it is unlikely to happen very soon.
